MiguelLZPF/decentralized-code-trust
# Develop (#6)

* npm publications and package.json updated

* Feat/get base updates (#2)

* secure action secret

* added base changes (tested)

* Fix/external types (#4)

* done and tested

* fix should nots with ganache

* add artifacts to repo

* try to fix test actions
